tests/chainer_tests/training_tests/extensions_tests/test_logreport.py

```diff
@@ -0,0 +1,60 @@
+import shutil
+import tempfile
+import unittest
+
+import mock
+
+from chainer import testing
+from chainer import training
+from chainer.training import extensions
+
+
+class TestLogReport(unittest.TestCase):
+
+    def setUp(self):
+        self.temp_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.temp_dir)
+
+    def test_trigger(self):
```

Comment:
Method names starts with `_` is for internal use, so I'm hesitating to write a unit test that directly calls `log_report._write`.
And making `_write()` to `write()` seems not so good, I'm expecting most users never call.

How about checking content of `{out_dir}/log` after `trainer.run()`?